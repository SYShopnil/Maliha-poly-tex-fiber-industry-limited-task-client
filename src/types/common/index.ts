export interface ICommonReturnData {
  message: string;
  status: number;
}

export enum EAuth {
  AuthTokenCookieName = "auth",
}

export enum EDataTestId {
  cLoginFormWithSubmit = "cLoginFormWithSubmit",
  cLogoutContainer = "cLogoutContainer",
  SHeaderMain = "SHeaderMain",
  SProfile = "SProfile",
}
