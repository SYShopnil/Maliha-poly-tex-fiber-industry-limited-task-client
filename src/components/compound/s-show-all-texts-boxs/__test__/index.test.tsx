import { render, screen } from "@testing-library/react";
import { EDataTestId } from "@src/types/common";
import { SShowAllTextBox } from "..";

describe("When Input Check Box Component Rendered", () => {
  it("Expect render successfully", async () => {
    render(await SShowAllTextBox());
    const myElement = screen.getByRole(EDataTestId.SShowAllTextBox);
    expect(myElement).toMatchSnapshot();
  });
});
