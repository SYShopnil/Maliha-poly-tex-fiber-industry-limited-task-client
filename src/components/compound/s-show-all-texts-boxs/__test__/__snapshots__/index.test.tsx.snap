// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Input Check Box Component Rendered Expect render successfully 1`] = `
<div
  role="SShowAllTextBox"
>
  <div
    class="flex justify-center items-center h-[50vh]"
    role="SLoading"
  >
    <p
      class="text-center font-extrabold text-2xl text-[#7F4D4F]"
    >
      Text Not Found Please Add...
    </p>
  </div>
</div>
`;
